Sources/ComposeUI/Scroll View/ComposeScrollView.swift

```diff
@@ -119,6 +124,12 @@ public struct ComposeScrollView<Content : View> : View {
     
 }
 
+public enum ComposeScrollPosition : Equatable {
```

Comment:
Compiler complained that this could not be a static property of ComposeScrollView. Let me know if this a good place and name for this.

Reply:
I think it's good for some near future at least